tests/test_ec2_manager.py

```diff
@@ -3,17 +3,85 @@
 import pytest
 
 from baram.ec2_manager import EC2Manager
+from baram.efs_manager import EFSManager
+from baram.sagemaker_manager import SagemakerManager
 
 
 @pytest.fixture()
 def em():
     return EC2Manager()
 
 
-def test_list_security_groups(em):
-    for g_id in em.list_security_groups():
-        print(g_id['GroupName'])
-    assert em.list_security_groups()
+def test_list_sgs(em):
+    pprint(em.list_sgs())
+    assert em.list_sgs()
+
+
+def test_list_unused_sg_ids(em):
+    sm = SagemakerManager()
+    redundant_sm_domain_ids = [domain['DomainId'] for domain in sm.list_domains()]
+
+    pprint(em.list_unused_sg_ids('NFS', redundant_sm_domain_ids))
+    assert type(em.list_unused_sg_ids()) == set
+
+
+def test_list_vpc_sg_eni_subnets(em):
+    result = em.list_vpc_sg_eni_subnets()
+    pprint(result)
+    assert type(result) == list
+    assert len(result) == len(em.list_sgs())
+    if len(result) != 0:
+        assert type(result[0]) == dict
+
+
+def test_get_sg_ids_with_vpc_id(em):
+    default_vpc_id = [vpc for vpc in em.list_vpcs() if vpc['IsDefault']][0]['VpcId']
+    pprint(em.get_sg_ids_with_vpc_id(default_vpc_id))
+
+
+def test_get_eni_with_sg_id(em):
+    sg_id = 'sg-0c41c743caf85d50b'
+    pprint(em.get_eni_with_sg_id(sg_id))
+
+
+def test_list_sg_relations(em):
+    pprint(em.list_sg_relations())
+    assert em.list_sg_relations()
+
+
+def test_get_related_sgs(em):
+    sg_id = 'sg-0cc39dfc07c29ba7e'
```

Comment:
remove specific ids in total code.

Reply:
This will be replaced with default value.